test-project/components/MainScene.brs

```diff
@@ -57,31 +57,16 @@ end function
 
 
 function configureLabel()
-  m.label.allFontSettings = {
-    header1Font: {
-      name: "font:MediumBoldSystemFont"
-      size: 130
-    }
-    header2Font: {
-      name: "font:MediumBoldSystemFont"
-      size: 60
-    }
-    header3Font: {
-      name: "font:MediumSystemFont"
-      size: 50
-    }
-    font: {
-      name: "font:MediumSystemFont"
-      size: 30
-    }
-    italicFont: {
-      name: "font:MediumBoldSystemFont"
-      size: 32
-    }
-    boldFont: {
-      name: "font:MediumSystemFont"
-      size: 34
-  } }
+  m.label.fontSettings = {
```

Comment:
I concur with showing the color setting usage here; but I'd like to not remove the other settings, as the point was to show how one configures all font settings in one place.